src/backend/metal/src/conversions.rs

```diff
@@ -52,7 +52,7 @@ pub fn map_format(format: Format) -> Option<MTLPixelFormat> {
         f::Rgba32Float       => RGBA32Float,
         f::D16Unorm          => Depth16Unorm,
         f::D32Float          => Depth32Float,
-        f::D24UnormS8Uint    => Depth24Unorm_Stencil8,
+        //f::D24UnormS8Uint    => Depth24Unorm_Stencil8, // !macOS
```

Comment:
The [docs state macOS 10.11+](https://developer.apple.com/documentation/metal/mtlpixelformat/mtlpixelformatdepth24unorm_stencil8), is that wrong?

Reply:
Interesting. I'm getting the validation error: pixel format 255 is invalid. Will follow-up in #2006